### PR TITLE
Fix ScaleHostEndToEndTest for ServiceBus

### PR DESCRIPTION
**Pull request by alrod**

### CODE REVIEW

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs [hunk 1]
```diff
@@ -102,7 +102,7 @@ public async Task<long> GetMessageCountAsync()
             }
             catch (Exception e)
             {
-                _logger.LogWarning($"Error querying for Service Bus {entityName} scale status: {e.Message}");
+                _logger.LogWarning($"Error querying for Service Bus {entityName} scale status", e);
```
Comment (JoshLove-msft) [inline sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs:105]: Is this change intended?

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ScaleHostEndToEndTests.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ public ScaleHostEndToEndTests() : base(isSession: false)
         }
 
         [Test]
-        [Ignore("Consistently failing in CI; awaiting investigation")]
+        //[Ignore("Consistently failing in CI; awaiting investigation")]
```
Comment (JoshLove-msft) [inline sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ScaleHostEndToEndTests.cs:34]: Delete

**Review by JoshLove-msft — APPROVED** [review]

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs [hunk 3]
```diff
@@ -96,13 +96,13 @@ public async Task<long> GetMessageCountAsync()
...
                 {
-                    _logger.LogWarning($"Connection string does not have 'Manage Claim' for {entityName} '{_entityPath}'. Unable to determine active message count.", ex);
+                    _logger.LogWarning(ex, $"Connection string does not have 'Manage Claim' for {entityName} '{_entityPath}'. Unable to determine active message count.");
                 }
                 throw ex;
             }
             catch (Exception e)
             {
-                _logger.LogWarning($"Error querying for Service Bus {entityName} scale status: {e.Message}");
+                _logger.LogWarning(e, $"Error querying for Service Bus {entityName} scale , stack: {e.StackTrace}");
```
Comment (JoshLove-msft) [inline sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs:105]: Why do we include the StackTrace in the message?

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs (hunk 1 above)
Comment (alrod) [reply sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs]: this is outdated, I fixed this by moving the exception to the first parameter.

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs (hunk 3 above)
Comment (alrod) [reply sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs:105]: this is outdated, I fixed this by moving the exception to the first parameter.

Comment (alrod) [reply sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs]: removed
